sources/packages/backend/libs/test-utils/src/factories/student-account-application.ts

```diff
@@ -0,0 +1,31 @@
+import { StudentAccountApplication, User } from "@sims/sims-db";
+
+/**
+ * Creates a fake student account application.
+ * @param relations dependencies:
+ * - `user` student that requested an account creation. It must be persisted already to database.
+ * - `assessedBy` ministry user that approved or denied the student account application. It must be persisted already to database.
+ * @param options options:
+ * - `initialValue` values to be used instead of the default ones.
+ * @returns a fake student account application ready to be persisted.
+ */
+export function createFakeStudentAccountApplication(
+  relations: {
+    user: User;
+    assessedBy?: User;
+  },
+  options?: {
+    initialValues?: Partial<StudentAccountApplication>;
```

Comment:
Minor difference between the parameter and the comment `initialValue` vs `initialValues`.

Reply:
updated